src/soundio/soundmanagerutil.h

```diff
@@ -163,46 +163,50 @@ class AudioInputBuffer : public AudioInput {
 
 
 class AudioSource {
-public:
-  virtual const CSAMPLE* buffer(const AudioOutput& output) const = 0;
+  public:
+    virtual ~AudioSource() = default;
```

Comment:
this forces the generation of the destructor, are you sure this is required?

Reply:
This is required to silence the warning. It does not fix an actual bug, because by now we do not delete the derived class form an AudioSource*.  

The destructor is there in any case, the change puts it into the V-Table, that it can be overridden.